pkg/controller/podgroup.go

```diff
@@ -167,11 +170,20 @@ func (v *VolcanoCtrl) decoratePodTemplateSpec(pts *corev1.PodTemplateSpec, mpiJo
 	pts.Annotations[volcanov1beta1.KubeGroupNameAnnotationKey] = mpiJobName
 }
 
-func (v *VolcanoCtrl) calculatePGMinResources(_ *int32, mpiJob *kubeflow.MPIJob) *corev1.ResourceList {
-	if schedPolicy := mpiJob.Spec.RunPolicy.SchedulingPolicy; schedPolicy != nil {
+// calculatePGMinResources calculate minResources for volcano podGroup
+// default to total replicas
+// ret: https://github.com/volcano-sh/volcano/blob/1933d46bdc4434772518ebb74c4281671ddeffa1/pkg/webhooks/admission/jobs/mutate/mutate_job.go#L168
+// ref: https://github.com/volcano-sh/volcano/blob/1933d46bdc4434772518ebb74c4281671ddeffa1/pkg/controllers/job/job_controller_actions.go#L761
+func (v *VolcanoCtrl) calculatePGMinResources(minMember *int32, mpiJob *kubeflow.MPIJob) *corev1.ResourceList {
```

Comment:
If we don't set minResources, does the volcano automatically calculate resources and fill out the PodGroup?

Reply:
I test this it will not set minResources when it is empty. it will automatically calculate minResources just in a vcJob.

Reply:
I have raised an issue a few months ago about this to keep correspding with vcjob when handle other podgroups. https://github.com/volcano-sh/volcano/issues/2791

Reply:
How about the latest?

Reply:
Thanks. I left a comment.